uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1015.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+trait CRM_HRCore_Upgrader_Steps_1015 {
+  
+  /**
+   * Update is_locked value for phone type option group
+   *
+   * @return bool
+   */
+  public function upgrade_1012() {
+    $data = [
+      'id' => '$value.id',
+      'is_locked' => 1,
+      'is_active' => 1
```

Comment:
Maybe you could add a comment about the bug that requires `is_active`

Reply:
That is a good idea. Thanks.